### PR TITLE
Fix secure inclusion  section. Source: AlCalzone

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -845,11 +845,13 @@ Names set in Home Assistant will not import into Z-Wave JS UI.
...
 ### Should I use `Secure Inclusion`?
 
-That depends. There are two generations of Z-Wave security, S0, and S2.
+That depends. There are two generations of Z-Wave encryption, Security S0, and Security S2. Both provide encryption and allow detecting packet corruption.
 
-S0 security imposes significant additional traffic on your mesh and is recommended only for devices that require security, such as door locks.
+Security S0 imposes significant additional traffic on your mesh and is recommended only for older devices that do not support Security S2 but require encryption to work, such as door locks.
 
-S2 security does not impose additional network traffic and provides additional benefits, such as detecting packet corruption. By default, Z-Wave attempts S2 security during inclusion if supported, falling back to S0 security only when necessary.
+S2 security does not impose additional network traffic and provides additional benefits. For example, end devices using S2 require the hub to report whether it has received and understood their reports.
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown]:
```suggestion
Security S2 does not impose additional network traffic and provides additional benefits. For example, end devices using S2 require the hub to report whether it has received and understood their reports.
```
this is the "official" name

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -845,11 +845,13 @@ Names set in Home Assistant will not import into Z-Wave JS UI.
...
+
+By default, Z-Wave prefers S2 security, if supported. S0 is used only when absolutely necessary.
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown]: dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: source/_integrations/zwave_js.markdown (hunk 2 above)
Comment (c0ffeeca7) [inline source/_integrations/zwave_js.markdown]:
```suggestion
By default, Z-Wave prefers Security S2, if supported. Security S0 is used only when absolutely necessary.
```

**Review by AlCalzone — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**